hyperspy/axes.py

```diff
@@ -559,22 +559,50 @@ def value2index(self, value, rounding=round):
         ------
         ValueError
```

Comment:
Can you harmonize the docstrings of the two value2index functions (also Parameters section and first line) - or define a common string that is used in both.

Reply:
It should be pretty harmonised now